packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -261,6 +262,27 @@ const DropdownBase = kind({
 	},
 
 	computed: {
+		children: ({children, selected}) => {
```

Comment:
Do we need to account for passing `data-webos-voice-disabled` on to the children?

Reply:
Good question. Agreed on following up separately.